d2common/d2data/d2datadict/unique_appellation.go

```diff
@@ -0,0 +1,106 @@
+package d2datadict
+
+import (
+	"log"
+
+	"github.com/OpenDiablo2/OpenDiablo2/d2common"
+)
+
+type UniqueAppellationRecord struct {
+	// The title
+	Name string
+
+	// The following booleans assign the title for a specific monster type
+	MonType1  bool
```

Comment:
This is for the wrong version of the txt file (from d2data.mpq). There is a newer version in the patch file (patch_d2.mpq) and the only column in that file is `Name`